lib/index.js

```diff
@@ -218,7 +222,11 @@
           if (originCallback) {
             originCallback(req.headers.origin, function (err2, origin) {
               if (err2 || !origin) {
```

Comment:
If `err2` is defined, it must always be passed to `next`, otherwise the error will be lost. The error argument is part of the standard Node.js callback contract.

Reply:
Done.